pkg/scheduler/cache/cache.go

```diff
@@ -182,8 +184,41 @@ func (su *defaultStatusUpdater) UpdatePodCondition(pod *v1.Pod, condition *v1.Po
 }
 
 // UpdatePodGroup will Update pod with podCondition
-func (su *defaultStatusUpdater) UpdatePodGroup(pg *v1alpha1.PodGroup) (*v1alpha1.PodGroup, error) {
-	return su.kbclient.SchedulingV1alpha1().PodGroups(pg.Namespace).Update(pg)
+func (su *defaultStatusUpdater) UpdatePodGroup(pg *api.PodGroup) (*api.PodGroup, error) {
+	if pg.Version == api.PodGroupVersionV1Alpha1 {
+		podGroup, err := api.ConvertPodGroupInfoToV1Alpha(pg)
+		if err != nil {
+			glog.Errorf("Error while converting PodGroup to v1alpha1.PodGroup with error: %v", err)
+		}
+		updated, err := su.kbclient.SchedulingV1alpha1().PodGroups(podGroup.Namespace).Update(podGroup)
+		if err != nil {
+			glog.Errorf("Error while updating podgroup with error: %v", err)
+		}
+		podGroupInfo, err := api.ConvertV1Alpha1ToPodGroupInfo(updated)
```

Comment:
I am concerned with the performance, since there are 2Xmarshal, 2X unmarshal in one update.